tasks/ast_tools/src/generators/typescript.rs

```diff
@@ -27,20 +27,30 @@ impl Generator for TypescriptGenerator {
         let estree_derive_id = codegen.get_derive_id_by_name("ESTree");
 
         let mut code = String::new();
+        let mut ast_node_names: Vec<String> = vec![];
         for type_def in &schema.types {
             if type_def.generates_derive(estree_derive_id) {
-                generate_ts_type_def(type_def, &mut code, schema);
+                generate_ts_type_def(type_def, &mut code, schema, &mut ast_node_names);
             }
         }
 
+        // manually append `FormalParameterRest`, which is generated via `add_ts_def`.
```

Comment:
The nittiest of nits!

```suggestion
        // Manually append `FormalParameterRest`, which is generated via `add_ts_def`
```